proto/regen/data/v1/events.proto

```diff
@@ -6,16 +6,16 @@ option go_package = "github.com/regen-network/regen-ledger/x/data";
 
 // EventAnchor is an event emitted when data is anchored on chain.
 message EventAnchor {
-  // iri is the data IRI
+  // iri is the IRI of the data anchored on chain.
   string iri = 1;
 }
 
 // EventAttest is an event emitted when data is attested to on chain.
 message EventAttest {
-  // iri is the data IRI
+  // iri is the IRI of the data attested to.
```

Comment:
```suggestion
  // IRI of the data attested to.
```

Reply:
This is not the convention for writing comments. We always start with the name of the field.